jspoon/src/main/java/pl/droidsonroids/jspoon/Utils.java

```diff
@@ -49,6 +58,23 @@
         }
     }
 
+    /**
+     * Returns a list of all fields, including inherited, starting from the upper most superclass.
+     * @param target the target subclass
+     * @return all declared fields
+     */
+    public static List<Field> getAllDeclaredFields(Class<?> target) {
+        List<Field> declaredFields = new ArrayList<>();
+        if (target.getSuperclass() == null || target == Object.class || target.isInterface()){
```

Comment:
Code can be reformatted, there should be a space before curly bracket.

Reply:
Done.